### PR TITLE
DC Expensing Amendment to Charge for ACK frames

### PR DESCRIPTION
**Pull request by lthiery**
Our current definition for how DCs are expensed for LoRaWAN traffic did not account for "empty" acknowledgement packets, ie: ACK frames that do not have FRMPayload or FOpts. I do not believe our intention was to make ACKs free, so this has been corrected.

In addition, article `0004` is renamed from the generic "data credits" to "expensing data credits for lorawan", as the article itself is not nearly as broad as the original title suggested.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ More details on process and how to participate to come.
 | 1  | [Longfi and LoRaWAN](https://github.com/helium/HIP/blob/master/0001-longfi-and-lorawan.md) | Merged |
 | 2  | [Sign miner](https://github.com/helium/HIP/blob/master/0002-sign-miner.md) | Merged |
 | 3  | [Over-the-air miner upates](https://github.com/helium/HIP/blob/master/0003-miner-update.md) | Merged |
-| 4  | [Data credits](https://github.com/helium/HIP/blob/master/0004-data-credits.md) | Merged |
+| 4  | [Exensing Data Credits for LoRaWAN Traffic](https://github.com/helium/HIP/blob/master/0004-data-credits.md) | Merged |
```
Comment (jamiew) [inline README.md:14]: Typo in `expensing` here

Comment (lthiery) [reply README.md]: Thanks for the catch, @jamiew

**Comment by jamiew** [comment]
IMHO this looks good to merge and move to formal `discussion` stage. Per #26 next step would be opening up a discussion issue and flagging for next community call

**Comment by jamiew** [comment]
Merged in, thanks for helping correct the HIP to its actual design now

**Pull request merged**